content/templates/_all.yml

```diff
@@ -14,6 +14,8 @@ home:
     - skip-link
     - buttons
     - headings
+    - card
+    - cta-link
```

Comment:
we can probably add search box in here too right?

https://design-system-starter.apps.b.cld.gov.au/